src/components/SearchBar.js

```diff
@@ -1,15 +1,46 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import { searchPhoto, onType } from '../actions';
 
-const SearchBar = () => {
-    return (
-        <div className="ui loading search">
-            <div className="ui icon input">
-                <input className="prompt" type="text" placeholder="Search..." />
-                <i className="search icon"></i>
-            </div>
-            <div className="results"></div>
-        </div>
-    );
+class SearchBar extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    onSearchChange = (event) => {
+        event.preventDefault();
+
+        this.props.onType(event.target.value);
+    }
+
+    onSearchSubmit = (event) => {
+        event.preventDefault();
+
+        this.props.searchPhoto(this.props.searchTerm);
+    }
+
+    render() {
+        return (
+            <form className="ui medium">
```

Comment:
I think its fine for now to use the the imported styles. But ultimate we want to move away from that. When working in an eCommerce environment you'll want to become very comfortable handling all of your own styles. I'll create a git issue to address this for later change.